ja/pagination.rst

```diff
@@ -0,0 +1,395 @@
+.. 
+    Pagination
+
+ページネーション
+##########
+
+.. 
+    php:namespace:: Cake\Controller\Component
+php:namespace::Cake\Controller\Component
+
+.. 
+    php:class:: PaginatorComponent
+php:class:: PaginatorComponent
+
+.. 
+    One of the main obstacles of creating flexible and user-friendly web
+    applications is designing an intuitive user interface. Many applications tend to
+    grow in size and complexity quickly, and designers and programmers alike find
+    they are unable to cope with displaying hundreds or thousands of records.
+    Refactoring takes time, and performance and user satisfaction can suffer.
+
+フレキシブルでかつユーザーフレンドリーなウェブアプリケーションを作成する際の主たる障害の一つとなるのが、直感的なユーザーインターフェイスです。多くのアプリケーションはすぐに巨大となりかつ複雑になり、デザイナーやプログラマーは、何百件、何千件ものレコードが表示されることに対応しきれなくなってきます。リファクタリングするには時間がかかり、パフォーマンスやユーザー満足度が犠牲になることが多いです。
+
+
+..   
+    Displaying a reasonable number of records per page has always been a critical
+    part of every application and used to cause many headaches for developers.
+    CakePHP eases the burden on the developer by providing a quick, easy way to
+    paginate data.
+
+一ページあたりに表示されるレコードの数を一定数に抑えることは、すべてのアプリケーションにとって重大な課題であり、ディベロッパーにとって頭の痛い問題でした。CakePHPは素早く、かつ簡単に、データをページ分けする方法を提供することで、ディベロパーへの負担を軽減させます。
+
+
+.. 
+    Pagination in CakePHP is offered by a Component in the controller, to make
+    building paginated queries easier. In the View
+    :php:class:`~Cake\\View\\Helper\\PaginatorHelper` is used to make the generation
+    of pagination links & buttons simple.
+
+CakePHPにおけるページネーションは、コントローラーにおけるコンポーネントによって提供され、ページ分けされたクエリーをより簡単にビルドできるようにします。ビューの中のView\Helper\PaginatorHelper は、ページネーションのリンクやボタンを作り出すことを容易にすることに使われます。
+
+
+.. 
+    Using Controller::paginate()
+Controller::paginate()の使用
```

Comment:
I think you want a newline between the comment and translated text so sphinx doesn't get confuse.